funder/commands/issue_and_activate.py

```diff
@@ -56,13 +93,16 @@ def issueAndActivateBounty(state, ipfsHash):
 def canUserFundBounty(state, ether, tokens=0):
     # check if user has enough ether
     # need to account for gas limit regardless, only add amount if bounty is funded using ether
-    amount = state.get('gas_limit')
+    # need to do two transactions, one to the bountiesNetwork and other as platform fees to gitcoin
+    amount = 2 * state.get('gas_limit')
```

Comment:
whats the deal with the `2 *`

Reply:
I thought that gas for each trx is being accounted for, but just saw that it is gas_limit. Will correct this.